app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    if (not isinstance(command, str) or len(command) < 3
+            or command[:3] != "cp "):
+        return
+
+    cmd_list = command.strip().split(" ")
+    if len(cmd_list) < 3 or cmd_list[-2] == cmd_list[-1]:
+        return
+
+    with open(cmd_list[-2], "r") as file_in,\
+            open(cmd_list[-1], "w") as file_out:
```

Comment:
The 'cp' command usually overwrites the destination file if it already exists. But in Python, trying to open a non-existing file for reading will raise a FileNotFoundError. It's a good practice to handle such exceptions and provide a clear error message. Also, consider checking if the source and destination files are not the same, because trying to copy a file onto itself will result in an empty file.